Core/LocalAdmin.cs

```diff
@@ -181,47 +181,59 @@ internal async Task Start(string[] args)
         {
             await LoadJsonOrTerminate();
 
+            var autoEula = false;
+
             if (DataJson!.EulaAccepted == null)
             {
-                ConsoleUtil.WriteLine($"Welcome to LocalAdmin version {VersionString}!", ConsoleColor.Cyan);
-                ConsoleUtil.WriteLine("Before starting please read and accept the SCP:SL EULA.", ConsoleColor.Cyan);
-                ConsoleUtil.WriteLine("You can find it on the following website: https://link.scpslgame.com/eula", ConsoleColor.Cyan);
-                ConsoleUtil.WriteLine("", ConsoleColor.Cyan);
-                ConsoleUtil.WriteLine("Do you accept the EULA? [yes/no]", ConsoleColor.Cyan);
+                if (args.Contains("--acceptEULA", StringComparer.Ordinal) ||
+                    Environment.GetEnvironmentVariable("SLLA_EULA")?.ToUpperInvariant() is "1" or "TRUE")
```

Comment:
Didn't notice it before, sorry, but env var should be renamed as well. I mentioned it in the comment before. Env var name (just like param name) should be clear that it accepts the EULA.

Don't forget to update the readme.md as well.

Reply:
np